sass/_elements.scss

```diff
@@ -0,0 +1,91 @@
+@import 'var';
+
+body {
+    font-size: 62.5%;
+    font-family:'Overpass', sans-serif;
+    font-weight: 200;
+    margin: 0 auto;
+    width: 100%;
+    line-height: 20px;
+    color: $dark-grey;
+}
+h1 {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 0px 0px;
+    width: 163px;
+}
+h2 {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 25px 0px;
+    font-family: 'Overpass', sans-serif;
+    font-size: 4.4em;
+    font-weight: 600;
+    color: $blue;
+    line-height: 55px
+}
+h3 {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 0px 0px;
+    font-family: 'Roboto', sans-serif;
+    font-size: 1.8em;
+    font-weight: 400;
+}
+p {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 0px 0px;
+    font-size: 1.3em;
+    margin-bottom: 20px
+}
+ul {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 0px 0px;
+    list-style-type: none;
+}
+li {
+    padding: 0px 0px 0px 0px;
+    margin: 0px 0px 0px 0px;
+    font-size: 1.3em;
+    list-style: none;
+}
+a {
+    text-decoration: none;
+}
+img {
+    width: 100%;
+    height: auto;
+}
+header {
+    margin-bottom: 70px;
+    background-image: url("pics/car-headlight.png");
```

Comment:
This path should work if you add `../` to the beginning so that you back out a folder. Probably a remnant from when this was in the root folder but now we've moved into the `sass/` and `css/` folders.